README.md

```diff
@@ -0,0 +1,150 @@
+<p align="center">
+  <a href="https://pennylane.ai">
+    <img width=80% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/pennylane_thin.png">
+  </a>
+</p>
+
+<p align="center">
+  <!-- Tests (GitHub actions) -->
+  <a href="https://github.com/PennyLaneAI/pennylane/actions?query=workflow%3ATests">
+    <img src="https://img.shields.io/github/workflow/status/PennyLaneAI/pennylane/Tests/master?logo=github&style=flat-square" />
+  </a>
+  <!-- CodeCov -->
+  <a href="https://codecov.io/gh/PennyLaneAI/pennylane">
+    <img src="https://img.shields.io/codecov/c/github/PennyLaneAI/pennylane/master.svg?logo=codecov&style=flat-square" />
+  </a>
+  <!-- ReadTheDocs -->
+  <a href="https://pennylane.readthedocs.io">
+    <img src="https://img.shields.io/readthedocs/pennylane.svg?logo=read-the-docs&style=flat-square" />
+  </a>
+  <!-- PyPI -->
+  <a href="https://pypi.org/project/PennyLane">
+    <img src="https://img.shields.io/pypi/v/PennyLane.svg?style=flat-square" />
+  </a>
+  <!-- Forum -->
+  <a href="https://discuss.pennylane.ai">
+    <img src="https://img.shields.io/discourse/https/discuss.pennylane.ai/posts.svg?logo=discourse&style=flat-square" />
+  </a>
+  <!-- License -->
+  <a href="https://www.apache.org/licenses/LICENSE-2.0">
+    <img src="https://img.shields.io/pypi/l/PennyLane.svg?logo=apache&style=flat-square" />
+  </a>
+</p>
+
+<p align="center">
+  <a href="https://pennylane.ai">PennyLane</a> is a cross-platform Python library for <a
+  href="https://en.wikipedia.org/wiki/Differentiable_programming)">differentiable
+  programming</a> of quantum computers.
+</p>
+
+<p align="center">
+  <strong>Train a quantum computer the same way as a neural network.</strong>
+  <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/update-readme/doc/_static/readme/code.png" width="800px">
+</p>
+
+
+## Key Features
+
+- *Brings quantum hardware to machine learning*. Support for hybrid quantum and classical models;
+  connect quantum hardware with **PyTorch**, **TensorFlow**, **Keras**, and **NumPy**.
+
+- *Device independent*. The same quantum circuit can be run on different quantum backends. Install
+  [plugins](https://pennylane.ai/plugins.html) to access even more devices, including Strawberry
+  Fields, IBM Q, Google Cirq, Rigetti Forest, Qulacs, Pasqal, Honeywell, and more.

```

Comment:
Should we bold these?

Reply:
Up to you, I worry there are too many now for it to look good 🤔

Reply:
let's give it a shot and see how it looks

Reply:
```suggestion
  [plugins](https://pennylane.ai/plugins.html) to access even more devices, including **Strawberry
  Fields**, **IBM Q**, **Google Cirq**, **Rigetti Forest**, **Qulacs**, **Pasqal**, **Honeywell**, and more.
```